helm/prometheus-meta-operator/templates/configmap.yaml

```
@@ -26,9 +26,15 @@ data:
         storage:
           createPVC: {{ and .Values.Installation.V1.Monitoring.Prometheus.Storage.Enabled .Values.Installation.V1.Monitoring.Prometheus.Storage.CreatePVC }}
           size: 20Gi
+      security:
+        letsencrypt:
+          enabled: {{ .Values.Installation.V1.Monitoring.Prometheus.Letsencrypt }}
+        restrictedAccess: 
+          enabled: {{ .Values.Installation.V1.Security.RestrictAccess.Enabled }}
+          subnets: {{ .Values.Installation.V1.Security.Subnet.VPN }}
       provider:
         kind: {{ .Values.Installation.V1.Provider.Kind }}
       installation:
         name: {{ .Values.Installation.V1.Name }}
       vault:
-        host: {{ .Values.Installation.V1.Auth.Vault.Host }}
+        host: {{ .Values.Installation.V1.Auth.Vault.Host }}
```

I know, this is very much a nitpick but no newline at the end of file triggers my OCD 😉 

```suggestion
        host: {{ .Values.Installation.V1.Auth.Vault.Host }}

```